src/components/v5/frame/NavigationSidebar/NavigationSidebarContent.tsx

```diff
@@ -0,0 +1,224 @@
+              </button>
+            </div>
+            {isTablet ? (
+              <>
+                <HamburgerButton
+                  onClick={() => {
+                    toggleMenu();
+                    setOpenItemIndex(-1);
+                  }}
+                  isOpen={isMenuOpen}
+                  label={hamburgerLabel}
+                />
+                <NavigationSidebarMobileContentWrapper
+                  mobileBottomContent={mobileBottomContent}
+                  isOpen={isMenuOpen && !!openItemIndex}
+                >
+                  {mainMenu}
+                </NavigationSidebarMobileContentWrapper>
+                <NavigationSidebarMobileContentWrapper
+                  mobileBottomContent={mobileBottomContent}
+                  isOpen={!openItemIndex}
+                >
+                  <NavigationSidebarSecondLevel {...colonySwitcherContent} />
+                </NavigationSidebarMobileContentWrapper>
+              </>
+            ) : (
+              mainMenu
+            )}
+          </div>
+          {additionalMobileContent && isTablet && (
+            <div>{additionalMobileContent}</div>
+          )}
+        </div>
+        {logo && !isTablet && <div className="mt-auto">{logo}</div>}
+      </div>
+      <AnimatePresence>
+        {isSecondLevelMenuOpen && !isTablet && (
+          <>
+            <motion.div
+              variants={secondLevelWrapperAnimation}
+              initial="hidden"
+              animate="visible"
+              exit="hidden"
+              className="h-full overflow-hidden"
+            >
+              <motion.div
+                variants={secondLevelContentAnimation}
+                className="h-full"
+              >
+                <NavigationSidebarSecondLevel
+                  {...(openItemIndex
+                    ? activeMainMenuItem?.secondLevelMenuProps
+                    : colonySwitcherContent)}
+                  onArrowClick={
+                    activeMainMenuItem?.relatedActionsProps?.items.length
+                      ? toggleThirdLevelMenu
+                      : undefined
+                  }
+                  isExpanded={
+                    activeMainMenuItem?.relatedActionsProps?.items.length
+                      ? isThirdLevelMenuOpen
+                      : false
+                  }
+                />
+              </motion.div>
+            </motion.div>
+            {!!activeMainMenuItem?.relatedActionsProps && (
+              <motion.div
+                variants={thirdLevelWrapperAnimation}
+                initial="hidden"
+                exit="hidden"
+                animate={isThirdLevelMenuOpen ? 'visible' : 'hidden'}
+                className="absolute top-0 bottom-0 h-full left-[calc(100%-1rem)] -z-[1] overflow-hidden"
+              >
+                <div className="p-6 pl-10 pt-[1.8125rem] bg-gray-900 text-white rounded-r-lg h-full overflow-auto">
```

Comment:
Maybe it will better to add paddings like this? => `pb-6 pr-6 pl-10 pt-[1.8125rem]` instead of overwriting p-6?